webpack/published-components.js

```diff
@@ -85,7 +85,8 @@ const publishedComponents = {
   Avatar: "components/Avatar/Avatar.jsx",
   IconButton: "components/IconButton/IconButton.jsx",
   Slider: "components/Slider/Slider.jsx",
-  // hooks
+  // Don't remove next line
```

Comment:
aren't we need to add the same comment and plop maker to path: "src/hooks/index.js" too? why this one is working without it?

Reply:
It's all about the pattern. In hooks/index.js we just replace last \n with a new string and in published-components.js we should put the line in the middle of the file, so we are using plop marker - it's just the easiest way to write this regex
```
      {
        type: "append",
        path: "src/hooks/index.js",
        pattern: /(\n$)/gm,
        separator: "",
        template: `export { default as {{camelCase hookName}} } from "./{{camelCase hookName}}";\n`
      },
      {
        type: "append",
        path: "webpack/published-components.js",
        pattern: /(\/\/ plop_marker:published-hooks)/g,
        template: `  {{camelCase hookName}}: "hooks/{{camelCase hookName}}",`
      }
```